packages/recommend/src/methods/getRecommendations.ts

```diff
@@ -22,16 +22,14 @@ type GetRecommendations = (
 ) => WithRecommendMethods<RecommendClient>['getRecommendations'];
 
 export const getRecommendations: GetRecommendations = base => {
```

Comment:
Should we let users know that sending multiple queries with different model is not the expected behavior?

Reply:
It would work as expected. However, in our AI components, we won't mix multiple models.

Reply:
Ok!